modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_object.jsp

```diff
@@ -26,15 +26,13 @@ resourcePrimKey = ParamUtil.getLong(request, "resourcePrimKey");
 long parentResourceClassNameId = ParamUtil.getLong(request, "parentResourceClassNameId");
```

Comment:
What you mean by odd?? Look and feel is quite similiar than what we had before... Is is true that this would need some love but not in this task. 
This was before: 
![Uploading Screenshot 2022-10-28 at 10.22.46.png…]()

Reply:
I agreed to left the behavior of the move action unchanged.